lib/cli-engine.js

```diff
@@ -792,4 +792,12 @@ CLIEngine.prototype = {
 
 };
 
+/**
+ * Gets current version of ESLint
+ * @returns {string} version number
+ */
+CLIEngine.version = function() {
```

Comment:
Why not just `CLIEngine.version = pkg.version`? I'm not sure there's much value in making it a function.